Dockerfile

```diff
@@ -18,4 +18,4 @@ LABEL name="ocm-example-service" \
       vendor="Red Hat" \
       version="0.0.1" \
       summary="OCM Example Service API" \
-      description="OCM Example Service API"
+      description="OCM Example Service API"
```

Comment:
not OCM.

TRex is TAP Rest Example

Reply:
Planning to resolve as part of https://issues.redhat.com/browse/HYPBLD-162!